### PR TITLE
Release 1.0.4

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY

1. Generate docs
2. Link changelog to README
3. Add description in galaxy.yml
4. Generate changelog

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,19 @@ Vyos Collection Release Notes
...
+
+Minor Changes
+-------------
+
+- Moved intent testcases from integration suite to unit tests.
+
+Bugfixes
+--------
+
+- Added error pattern to the terminal plugin to handle change mode error seen in lag interfaces config.
```
Comment (samccann) [inline changelogs/CHANGELOG.rst:19]: for future changelogs - it helps if you add a link to either the issue or the PR that made the change.

**Comment by GomathiselviS** [comment]
recheck

**Comment by pabelanger** [comment]
We'll need to regenerate the changelog to inlcude recent black changes

**Comment by GomathiselviS** [comment]
Closing this PR and creating a new one, because of the reformatting issues (with latest version of Black)

**Pull request closed without merging**